src/core/issues-notes.js

```diff
@@ -51,7 +51,7 @@ function handleIssues(ins, ghIssues, conf) {
     // wrap
     if (!isInline) {
       const div = hyperHTML`<div class='${type +
-        (isFeatureAtRisk ? " atrisk" : "")}'></div>`;
+        (isFeatureAtRisk ? " atrisk" : "")}' role=${(type === "note"? "note" :"")} ></div>`;
```

Comment:
Might be a little easier on the eyes... also, setting it to `null` means that `role` attribute will not be there when not needed (instead of being `role=""`, which could mean something else):

```suggestion
        (isFeatureAtRisk ? " atrisk" : "")}'
        role='${type === "note"? "note" : null}'></div>`;
```